console/billing/how-to/use-billing-alerts.mdx

```diff
@@ -0,0 +1,63 @@
+---
+meta:
+  title: How to use Billing Alerts
+  description: This page explains how to use billing alerts
+content:
+  h1: How to use Billing Alerts
+  paragraph: This page explains how to use billing alerts
+tags: billing-alert alerts billing budget 
+dates:
+  validation: 2024-01-14
+  posted: 2024-01-14
+categories:
+  - console
+---
+
+The **Billing Alerts** functionality within the Scaleway console allows users to oversee and control their expenses efficiently. This feature allows you to create alerts that notify users when predefined budget thresholds are met. Your alerts can be seamlessly delivered through SMS, email, or API webhook channels.
+
+Your budget, denominated in euros, represents the limit for your expenses. Simultaneously, the threshold corresponds to a predetermined percentage of this budget. You have the autonomy to define both the budget and the threshold within the console interface.
+
+**For example:**
+
+  - If you have defined a budget of €1000, you may configure an email notification to be sent once you have consumed 50% of this limit. In this case, once your consumption gets to €500 you will receive an email informing you the limit threshold has been reached.
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+</Message>
+
+
+1. Go to the Billing page by clicking on **Billing** in the Organization drop-down menu of the Scaleway console.
```

Comment:
```suggestion
1. Access your [Billing page](https://console.scaleway.com/organization/billing) on the Scaleway console.
```

Reply:
> https://console.scaleway.com/organization/billing

There will be a new billing space/page that wont be accessible from the organization dashboard anymore.

Reply:
Indeed, the Billing tab will disappear from the Organization dashboard, but you can still access the Billing Space from the drop-down menu or the shortcut link (see screenshot). Jessie's suggestion works for me :) 
<img width="934" alt="Screenshot 2024-02-19 at 11 43 22" src="https://github.com/scaleway/docs-content/assets/124775951/0538148e-b405-48d6-9f8f-93e687d4bf9f">

Reply:
> Indeed, the Billing tab will disappear from the Organization dashboard, but you can still access the Billing Space from the drop-down menu or the shortcut link (see screenshot). Jessie's suggestion works for me :)

Ah okay.. the links stay the same? .. In that case I will change it for all the pages .. Thanks :)